RecoMET/METProducers/interface/HcalNoiseInfoProducer.h

```diff
@@ -130,8 +132,16 @@ namespace reco {
 
     uint32_t HcalAcceptSeverityLevel_;
     std::vector<int> HcalRecHitFlagsToBeExcluded_;
+
+    std::vector<int> LaserMonDetTypeList_;
```

Comment:
Class members should start with lowercase letters, uppercase initials are reserved to class names